lib/api/apiUtils/object/getReplicationInfo.js

```diff
@@ -1,33 +1,48 @@
 const s3config = require('../../../Config').config;
 
-function _getReplicationInfo(rule, replicationConfig, content, operationType) {
-    const storageClasses = rule.storageClass ?
-        rule.storageClass.split(',') : [];
+function _getBackend(objectMD, site) {
```

Comment:
I forgot the context for this one. Why would the backend not exist? And also, it's weird that a get call is updating the status.

Reply:
If it's the first time putting an object to be replicated it will push the new backend to the array. In the case where you are updating an object version (for example putting tags), we want to preserve the datastoreVersionId. 

I guess the comment is a bit misleading in the sense that it's not really "updating" the status, but rather preserving any existing object values in the `backends` array.